### PR TITLE
[BugFix] Validate query_timeout at analyze stage

### LINKED ISSUE #13695: [Bug] timeout can't be negative
### Steps to reproduce the behavior (Required)

cluster with 3fe

connect to a follower fe, and execute the following statements
```sh
set global query_timeout=360000000;
Query OK, 0 rows affected
Time: 0.009s

 set global query_timeout=360;
(1064, "timeout can't be negative")
```
### StarRocks version (Required)
 - You can get the StarRocks version by executing SQL `select current_version()`

```
+-------------------+
| current_version() |
+-------------------+
| UNKNOWN 84ce40cbd |
+-------------------+
```

### PR DESCRIPTION
**Pull request by liuyehcf**
## What type of PR is this：
- [x] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [ ] Doc
- [ ] Tool

## Which issues of this PR fixes ：

Fixes #13695

## Problem Summary(Required) ：

Validate `query_timeout` in analyze stage, with maximum value to `259200`

After this pr, you will get error message when setting `query_timeout` with an illegal value.

```
> set global query_timeout=360000000;
(1064, 'query_timeout must be equal or smaller than 259200.')
> set global query_timeout=0;
(1064, 'query_timeout must be equal or greater than 1.')
```

## Checklist:

- [x] I have added test cases for my bug fix or my new feature
- [ ] This pr will affect users' behaviors
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

### CODE REVIEW

**Review by kangkaisen — DISMISSED** [review]

File: fe/fe-core/src/main/java/com/starrocks/sql/ast/SetVar.java [hunk 1]
```diff
@@ -176,12 +185,15 @@ public void analyze() {
...
+    private void checkRangeLongVariable(String field, Long min, Long max) {
         String value = getResolvedExpression().getStringValue();
         try {
             long num = Long.parseLong(value);
-            if (num < 0) {
-                throw new SemanticException(field + " must be equal or greater than 0.");
+            if (min != null && num < min) {
+                throw new SemanticException(String.format("%s must be equal or greater than %d.", field, min));
+            }
+            if (max != null && num > max) {
```
Comment (HangyuanLiu) [inline fe/fe-core/src/main/java/com/starrocks/sql/ast/SetVar.java:195]: why not use Long.MAX_VALUE?

**Review by Seaven — DISMISSED** [review]

**Comment by wanpengfei-git** [comment]
run starrocks_admit_test

**Comment by liuyehcf** [comment]
run starrocks_fe_unittest

File: fe/fe-core/src/main/java/com/starrocks/sql/ast/SetVar.java (hunk 1 above)
Comment (liuyehcf) [reply fe/fe-core/src/main/java/com/starrocks/sql/ast/SetVar.java:195]: Different variables has different uppler limits

**Comment by liuyehcf** [comment]
run starrocks_admit_test

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:disappointed: **fail** : 12 / 17 (70.59%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/qe/LeaderOpExecutor.java|0|2|00.00%|[73, 74]|
|:large_blue_circle:|com/starrocks/sql/ast/SetVar.java|11|14|78.57%|[98, 169, 196]|
|:large_blue_circle:|com/starrocks/qe/SessionVariable.java|1|1|100.00%|[]|

**Review by kangkaisen — APPROVED** [review]

**Review by Seaven — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
run starrocks_admit_test

**Comment by liuyehcf** [comment]
run pr-title-check

**Comment by liuyehcf** [comment]
All cherry-pick prs:
https://github.com/StarRocks/starrocks/pull/13750
https://github.com/StarRocks/starrocks/pull/13753
https://github.com/StarRocks/starrocks/pull/13755
https://github.com/StarRocks/starrocks/pull/13756

**Pull request merged**